meinberlin/apps/kiezkasse/exports.py

```diff
@@ -32,3 +34,34 @@ def get_queryset(self):
             .annotate_comment_count()\
             .annotate_positive_rating_count()\
             .annotate_negative_rating_count()
+
+
+@register_export(_('Comments for kiezkasse proposals'))
+class ProposalCommentExportView(a4_export_mixins.ExportModelFieldsMixin,
+                                export_mixins.UserGeneratedContentExportMixin,
+                                a4_export_mixins.ItemExportWithLinkMixin,
+                                a4_export_mixins.ItemExportWithRatesMixin,
+                                export_mixins.ItemExportWithRepliesToMixin,
+                                a4_export_views.BaseItemExportView):
+
+    model = Comment
+
+    fields = ['id', 'comment', 'created']
+
+    def get_queryset(self):
+        comments = (Comment.objects.filter(
+                    kiezkasse_proposal__module=self.module) |
+                    Comment.objects.filter(
+                    parent_comment__kieszkasse_proposal__module=self.module))
+
+        return comments
+
+    def get_base_filename(self):
```

Comment:
I am thinking that we don't need that part, that is already implemented in BaseItemExportView: https://github.com/liqd/adhocracy4/blob/029bd5fb27236f9c1c042e32b126e41fe4f9a73e/adhocracy4/exports/views.py#L103